README.md

```diff
@@ -61,88 +59,51 @@ A Go implementation is developed and maintained by the [Berty](https://github.co
 
 <!-- tocstop -->
 
-## Usage
-
-Read the **[GETTING STARTED](https://github.com/orbitdb/orbit-db/blob/master/GUIDE.md)** guide for a quick tutorial on how to use OrbitDB.
-
-For a more in-depth tutorial and exploration of OrbitDB's architecture, please check out the **[OrbitDB Field Manual](https://github.com/orbitdb/field-manual)**.
-
-### Module with IPFS Instance
-
-If you're using `orbit-db` to develop **browser** or **Node.js** applications, use it as a module with the javascript instance of IPFS
-
-Install dependencies:
+## Installation
 
 ```
-npm install orbit-db ipfs
+npm install orbit-db
 ```
 
+## Usage
+
+If you're using `orbit-db` to develop **browser** or **Node.js** applications, use it as a module with the javascript instance of IPFS.
 
 ```javascript
 import IPFS from 'ipfs'
 import OrbitDB from 'orbit-db'
 
 ;(async function () {
   const ipfs = await IPFS.create()
-  const orbitdb = await OrbitDB.createInstance(ipfs)
+  const orbitdb = await OrbitDB({ ipfs })
 
-  // Create / Open a database
-  const db = await orbitdb.log("hello")
-  await db.load()
+  // Create / Open a database. Defaults to db type "events".
+  const db = await orbitdb.open("hello")
```

Comment:
I think an important bit here, that we didn't have in the previous example, is the use of addresses. We could add:
```
const address = db.address
console.log(address)
// "/orbitdb/hash"
// The above address can be used on another peer to open the same database
```